config/locales/manage_outcomes.en.yml

```diff
@@ -7,6 +7,7 @@ en:
         view_outcomes: View Outcomes
       courses_table:
         actions: Actions
+        assessment_coverage: Assessment Coverage
```

Comment:
@jdaries Are you okay with the progress bar header being "Assessment Coverage"? I think that more accurately describes it rather than just "Assessments".